src/actor/component.rs

```diff
@@ -159,6 +180,24 @@ impl Instance<'_> {
         operation: impl AsRef<str>,
         payload: Option<impl AsRef<[u8]>>,
     ) -> anyhow::Result<Result<Option<Vec<u8>>, String>> {
+        self.call_with_context(operation, payload, None::<Vec<u8>>)
+            .await
+    }
+
+    /// Invoke an operation on an [Instance] producing a result, where outermost error represents
+    /// a WebAssembly execution error and innermost - the component operation error
+    /// The `call_context` argument is an opaque byte array that can be used for additional
+    /// metadata around an actor call, like a parent span ID or invocation ID.
+    #[instrument(skip_all)]
+    pub async fn call_with_context(
+        &mut self,
+        operation: impl AsRef<str>,
+        payload: Option<impl AsRef<[u8]>>,
+        call_context: Option<impl AsRef<[u8]>>,
```

Comment:
```suggestion
        call_context: impl AsRef<[u8]>,
```

`call_with_context` should always take a context